__tests__/Lens.test.js

```diff
@@ -156,4 +156,12 @@ describe('Lens', () => {
   test('should sequence lenses correctly', () => {
     expect(toArray(sequence('id', 'name'), user)).toEqual([1, 'Flavio'])
   })
+
+  test('useState-like lens works', () => {
+    var x = 1
```

Comment:
```suggestion
    let x = 1
```

Reply:
The point here is being able to check that the value has changed, so we really need `var`.

Reply:
You should *never* use `var` now in modern JS, if you want mutable variables, use `let` 😉